### PR TITLE
Add contact links to issue templates

### PR DESCRIPTION
**Pull request by stevehobbsdev**
- Add contact links to issues screen
- Add FAQ page

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -1 +1,14 @@
 blank_issues_enabled: false
+contact_links:
+  - name: Auth0 Community
+    url: https://community.auth0.com/tags/c/sdks/5
```
Comment (frederikprijck) [inline .github/ISSUE_TEMPLATE/config.yml]: Is this supposed to be a public available URL?

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -1 +1,14 @@
...
+    about: Discuss this SDK in the Auth0 Community forums
+  - name: FAQ
+    url: https://github.com/auth0/auth0-angular/blob/master/FAQ.md
+    about: Read the FAQ to get answers to common issues
+  - name: SDK API Documentation
+    url: https://auth0.github.io/auth0-angular/
+    about: Read the API documentation for this SDK
+  - name: Library Documentation
+    url: https://auth0.com/docs/libraries/auth0-angular
```
Comment (frederikprijck) [inline .github/ISSUE_TEMPLATE/config.yml]: This gives a 404

File: FAQ.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+## 2. User is not logged in after successful sign in with redirect
+
+If after successfully logging in, your user returns to your spa and is still not authenticated, do _not_ refresh the page - go to the Network tab on Chrome and confirm that the POST to `oauth/token` resulted in an error `401 Unauthorized`. If this is the case, your tenant is most likely misconfigured. Go to your **Application Properties** in your application's settings in the [Auth0 Dashboard](https://manage.auth0.com) and make sure that `Application Type` is set to `Single Page Application` and `Token Endpoint Authentication Method` is set to `None` (**Note:** there is a known issue with the Auth0 "Default App", if you are unable to set `Token Endpoint Authentication Method` to `None`, create a new Application of type `Single Page Application` or see the advice in [auth0-react/issues/93](https://github.com/auth0/auth0-react/issues/93#issuecomment-673431605))
```
Comment (frederikprijck) [inline FAQ.md]: Should we use `SPA` instead of `spa`?

**Review by frederikprijck — CHANGES_REQUESTED** [review]

File: FAQ.md (hunk 3 above)
Comment (stevehobbsdev) [reply FAQ.md]: Yes - corrected

File: .github/ISSUE_TEMPLATE/config.yml (hunk 2 above)
Comment (stevehobbsdev) [reply .github/ISSUE_TEMPLATE/config.yml]: Corrected

File: .github/ISSUE_TEMPLATE/config.yml (hunk 1 above)
Comment (stevehobbsdev) [reply .github/ISSUE_TEMPLATE/config.yml]: Corrected

**Review by frederikprijck — APPROVED** [review]

**Pull request merged**